README.md

```diff
@@ -1,7 +1,47 @@
 # MsgPack.Light
-MsgPack.Light is a lightweight http://msgpack.org/ format implementation.
+MsgPack.Light is a lightweight [MsgPack](http://msgpack.org/) format implementation.
 
-# Build statuses for master branch
+## Key features
+* Performance
+* .Net core compatability
+* Extensibility
+* Simple usage
+
+## Install
+Simpliest way to start using MsgPack.Light is to install [NuGet package](https://www.nuget.org/packages/MsgPack.Light/). To do that, run the following command in the  Package Manager Console:
```

Comment:
Don't use VS-specific workflow. Just say: install it from nuget.

Developers are not so clueless usually. E.g. I install packages by editing project.json file manually.